### PR TITLE
scripts/updatedep: work around 'testify/assert', remove 'etcd-top'

### PR DESCRIPTION
**Pull request by gyuho**
'glide vc --no-tests' flag removes 'testify/assert' deps
in v2 client. Until we deprecate v2 tests, just copy the
necessary files as workaround.

And remove all legacy packages in glide.yaml on sub-dependency.
They were added when we migrated from godep. glide will handle
it automatically with glide.lock file.

And remove '--skip-tests' flags in case we add dependencies
in test files.

/cc @xiang90 @heyitsanthony

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -147,8 +151,11 @@ imports:
...
 testImports:
-- name: github.com/stretchr/testify
-  version: ""
-  repo: https://github.com/stretchr/testify.git
+- name: github.com/davecgh/go-spew
+  version: 6d212800a42e8ab5c146b8ace3490ee17e5225f9
   subpackages:
-  - assert
+  - spew
+- name: github.com/pmezard/go-difflib
```
Comment (heyitsanthony) [inline glide.lock]: do you know why it's adding this (and go-spew) to the vendor deps but isn't adding it to the vendor directory?

Comment (gyuho) [inline glide.lock]: nvm. I was wrong. Will investigate why.

**Comment by gyuho** [comment]
@xiang90 @heyitsanthony All fixed. PTAL.

According to current `glide.lock` in master branch, the `testify/assert` wasn't vendored correctly (version string is empty in `glide.lock`). Think the version was missing when we first migrated off from `godep`, and somehow we were running the old version of testify, which works fine as well.

Now that `testify/assert` is correctly vendor-ed, `glide` picks up `testify/assert`'s sub-dependencies, which is why this PR has +6k LOC changes in this PR. They will eventually disappear when we deprecate v2.

**Comment by gyuho** [comment]
@heyitsanthony @xiang90 @fanminshi 

I would like to get this merged before we do grpc dependency bump up. Otherwise current vendor script needs manual reverting work.

I removed dependencies in `tools/etcd-top` since we only partially vendors anyway with `pcap` removed.

Any objection?

**Comment by heyitsanthony** [comment]
seems like etcd-top is more trouble than it's worth if it even needs a glide workaround; I'm fine with getting rid of it at this point

**Comment by gyuho** [comment]
Yeah... is there any way that we can have this outside of etcd? This is blocking our CI and vendoring work.

Defer to @xiang90 @spacejam (original author)

**Comment by gyuho** [comment]
@spacejam We are dropping `etcd-top` to move forward on CI issues.

Please let us know if you have any other concerns. Thanks.

**Pull request merged**